lib/ui.rb

```diff
@@ -2,13 +2,17 @@
 
 class UI
```

Comment:
The only thing I could think of was [this](http://octopusonalightbeam.tk/journal/2016/01/26/day074.html), but I also want to extract stuff from that class into smaller classes.